.changeset/fair-cats-lick.md

```diff
@@ -0,0 +1,16 @@
+---
+"@saleor/react-hook-form-macaw": patch
+"saleor-app-emails-and-messages": patch
+"saleor-app-data-importer": patch
+"saleor-app-products-feed": patch
+"@saleor/apps-shared": patch
+"saleor-app-invoices": patch
+"@saleor/e2e": patch
+"saleor-app-search": patch
+"@saleor/apps-ui": patch
+"saleor-app-taxes": patch
+"saleor-app-cms": patch
+"saleor-app-crm": patch
+---
+
+Update Zod to 3.21.4
```

Comment:
For the app itself I will skip changelog since this is the first release. Its not an "update" but the new app so there is nothing to compare to.

I will ensure other touched packages are described